.github/pull_request_template.md

```diff
@@ -13,7 +13,7 @@ Describe the original problem and the changes made on this PR.
 * JIRA: https://zendesk.atlassian.net/browse/AF-XXX
 
 ### Risks
-* [HIGH | medium | low] Does it work across browsers (including IE!)?
```

Comment:
As these changes are unrelated, it's preferable to split them out into their own PR (even if that PR is really small).

As an aside, "Does it work in the different products (Support, Chat, Connect)?" is probably also not relevant to ZAT

Reply:
Sounds good